github/package-lock.json

```diff
@@ -551,29 +609,49 @@
             }
         },
         "@rocket.chat/apps-engine": {
-            "version": "1.32.0",
-            "resolved": "https://registry.npmjs.org/@rocket.chat/apps-engine/-/apps-engine-1.32.0.tgz",
-            "integrity": "sha512-K+5NJto1BfO0NFiJX048tAWJqNQjdHGg5YPB24MQ/cDq206hDPnJakihF02LgardAQShIK3TIw2MnKgX1CB2/w==",
+            "version": "1.38.0",
+            "resolved": "https://registry.npmjs.org/@rocket.chat/apps-engine/-/apps-engine-1.38.0.tgz",
+            "integrity": "sha512-qY1LRggfZNieOkxDBuEnaieZK6gHfA2tNLrqvv0xn7vgZWkJlWvfcuhPbfarV14q3tuEJgnqf0uIoqia3+WJFQ==",
             "dev": true,
             "requires": {
-                "adm-zip": "^0.4.9",
+                "adm-zip": "^0.5.9",
                 "cryptiles": "^4.1.3",
+                "jose": "^4.11.1",
                 "lodash.clonedeep": "^4.5.0",
-                "semver": "^5.5.0",
+                "semver": "^5.7.1",
                 "stack-trace": "0.0.10",
-                "uuid": "^3.2.1"
+                "uuid": "^3.4.0",
+                "vm2": "^3.9.11"
             }
         },
+        "@rocket.chat/ui-kit": {
+            "version": "0.31.22",
+            "resolved": "https://registry.npmjs.org/@rocket.chat/ui-kit/-/ui-kit-0.31.22.tgz",
+            "integrity": "sha512-CH7kBVBsAql57DwDMOpMvj8lp+23muEqiaQ4x0cr0CV/HjmOBBiclzJ2wvh7ab7KEkABIhDK8dwSreNgk0LiJQ==",
+            "dev": true
+        },
```

Comment:
Reason for having `1.38.0`: 
- Lets say we have X dependency version ^1.36.0 so watchout the character ->`^` here if the dependency has new minor release(s) then package-lock.json will get updated when we run the npm install and its allowed to installed higher version having minor and patch changes.

- Also previously we were having the version of `1.32.0` instead of `1.36.0`. So there would be time when latest release would be 1.36.0. now to note when we updated the version of this app this PR https://github.com/RocketChat/Apps.Github22/pull/44 we forgot to change the package-lock.json meaning we forgot to run the `npm install` to synchronize the package.json and package-lock.json.

- There is one concept of installing dependency from package.json or package-lock.json:
- Currently to install dependency we have npm install in docs which means that it will install dependency looking at the package.json and is allowed to install the dependency having higher version in minor or patches.
- Further more to install dependency we have `npm ci` that will particularly look at the package-lock.json and won't change the package-lock.json.

### LGTM as per this dependency is concern

Reply:
### further checks:
- @Prince-Mendiratta have you deploy the app after this changes in dependency. It is working fine and able to deploy?

Reply:
Great! Everything happened as expected. LGTM!